### PR TITLE
Complete draft of documentation with developer guides.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**

### CODE REVIEW

File: docs/thanks.rst [hunk 1]
```diff
@@ -8,13 +8,28 @@ Cargo Sphinx was originally forked from the `cargo-release`_ project by
...
 The `Sphinx Bootstrap Theme`_ used in generating Cargo Sphinx documentation is
 by `Ryan Roemer`_. The documentation uses the Simplex theme from Bootswatch_.
 
 .. _Sphinx Bootstrap Theme: https://github.com/ryan-roemer/sphinx-bootstrap-theme
 .. _Ryan Roemer: https://github.com/ryan-roemer
 .. _Bootswatch: http://bootswatch.com
+
+README badging provided by `Travis CI`_, `Dependency CI`_, and `Shields.io`_.
+
+.. _Travis CI: http://travis-ci.org
```
Comment (passy) [inline docs/thanks.rst]: nit: Can be https

File: docs/thanks.rst [hunk 2]
```diff
@@ -8,13 +8,28 @@ Cargo Sphinx was originally forked from the `cargo-release`_ project by
...
+.. _Dependency CI: https://dependencyci.com
+.. _Shields.io: http://shields.io
```
Comment (passy) [inline docs/thanks.rst]: Same

**Comment by passy** [comment]
Fast! :shipit: of course!

**Comment by daithiocrualaoich** [comment]
Stalled on CI or this would be landed by now. :)

**Pull request merged**